zebra/rt_netlink.c

```diff
@@ -2325,6 +2333,18 @@ netlink_put_route_update_msg(struct nl_batch *bth, struct zebra_dplane_ctx *ctx)
 	if (dplane_ctx_get_op(ctx) == DPLANE_OP_ROUTE_DELETE) {
 		cmd = RTM_DELROUTE;
 	} else if (dplane_ctx_get_op(ctx) == DPLANE_OP_ROUTE_INSTALL) {
+		if (p->family == AF_INET6 && !v6_rr_semantics) {
```

Comment:
What situation are you seeing that leads you to the conclusion that FRR should do this?